widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java

```diff
@@ -32,7 +31,7 @@
  * Before this activity is launched, make sure that Glia Widgets SDK is set up correctly.
  * <p>
  * Required data that should be passed together with the Activity intent:
- * - {@link GliaWidgets#QUEUE_ID}: ID of the queue you would like to use for your engagements.
+ * - {@link GliaWidgets#QUEUE_ID}: ID of the queue you would like to use it for your engagements.
```

Comment:
I think "it" is not needed here because this sentence has subject already - "ID of the queue".